src/libs/actions/IOU.ts

```diff
@@ -8616,11 +8616,7 @@ function sendMoneyWithWallet(report: OnyxEntry<OnyxTypes.Report>, amount: number
     notifyNewAction(params.chatReportID, managerID);
 }
 
-function canApproveIOU(
-    iouReport: OnyxTypes.OnyxInputOrEntry<OnyxTypes.Report> | SearchReport,
-    policy: OnyxTypes.OnyxInputOrEntry<OnyxTypes.Policy> | SearchPolicy,
-    chatReportRNVP?: OnyxTypes.ReportNameValuePairs,
-) {
+function canApproveIOU(iouReport: OnyxTypes.OnyxInputOrEntry<OnyxTypes.Report> | SearchReport, policy: OnyxTypes.OnyxInputOrEntry<OnyxTypes.Policy> | SearchPolicy) {
```

Comment:
There wasn't any instance of `canApproveIOU()` that was passing more than two arguments, so I removed the third parameter entirely.

Trying to get rid of `getReportNameValuePairs()` is very difficult, so I'm going to slowly try to do it in other PRs. It's because `canApproveIOU()` is used in many places, so it will take many updates to be able to pass the `rNVPs` as an argument.

Reply:
Sounds good, I also think we could get a contributor to help with this if we make this [external](https://github.com/Expensify/App/issues/59961)? If there's a reason for it to be internal, I can also help with this